From a378a3564b894e0cbfe52241b391eabc0f6e8c5c Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@xxxxx.com>
Date: Wed, 23 Aug 2023 17:37:01 -0400
Subject: [PATCH] CONFIG_WERROR: Don't compile the kernel with warnings as errors

In some cases on this branch this is making builds fail.

Signed-off-by: Patrick Yavitz <pyavitz@xxxxx.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4231fadd7475..a76d3728ae01 100644
--- a/Makefile
+++ b/Makefile
@@ -804,7 +804,7 @@ stackp-flags-$(CONFIG_STACKPROTECTOR_STRONG)      := -fstack-protector-strong
 
 KBUILD_CFLAGS += $(stackp-flags-y)
 
-KBUILD_CFLAGS-$(CONFIG_WERROR) += -Werror
+KBUILD_CFLAGS-$(CONFIG_WERROR) +=
 KBUILD_CFLAGS += $(KBUILD_CFLAGS-y)
 
 ifdef CONFIG_CC_IS_CLANG
-- 
2.39.2

